Avatars/Avatars.css

```diff
@@ -0,0 +1,144 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700;900&display=swap');
+
+:root {
+    --primary-color : #c2004b;
+    --secondary-color : #eb679a;
+    --light-color : #ffeff5;
+    --dark-color : #72022d;
+    --subtitle-text-color : #0000008A;
+    --info-text-color : #757582;
+}
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Roboto;
```

Comment:
better to set font family on body element

Reply:
noted, thanks